lasagne/layers/conv.py

```diff
@@ -48,7 +49,7 @@ def __init__(self, incoming, num_filters, filter_length, stride=1,
             self.nonlinearity = nonlinearity
 
         self.num_filters = num_filters
-        self.filter_length = filter_length
+        self.filter_size = filter_size
```

Comment:
If we're going to use tuples everywhere, what do you think about standardizing on `filter_shape` and `pool_shape` instead? Seems like it would be more consistent with `input_shape` and `output_shape`.